### DISCUSSION TITLE
Classic theme Component name confusion while swizzling

### DISCUSSION THREAD

**Discussion by Devorein** [facebook/docusaurus#4153] [Q&A]
For swizzling components like Table of contents, I thought the right command was `npx docusaurus swizzle @docusaurus/theme-classic TOC`. Notice all uppercase `TOC`. I saw that the component was nested inside a folder named `TOC` in `node_modules/@docusaurus/theme-classic/lib/theme/TOC`. But after several failed attempts at swizzling, I thought maybe I was not typing the component name correctly. But, when I swizzled a component named `Toc` instead of `TOC` it worked perfectly. That was a mistake on my part but the naming convention of the folder played its part as well. Can I know how I can avoid such mistakes in the future? Is there a naming convention for the components somewhere? Thanks.

**Comment by slorber** [comment]
Hi,

It's not clear to me what command did you type, and what error did you get, and what are the full details of your env and Docusaurus version.

For me it seems to work, and suggest close matches in case of typo:

![image](https://user-images.githubusercontent.com/749374/106626834-63d02b80-6578-11eb-80eb-f52646e053a9.png)

You can also see a list of components by not providing the name:

![image](https://user-images.githubusercontent.com/749374/106626925-8104fa00-6578-11eb-86ba-6912d5743f72.png)

**Reply by Devorein** [reply]
Hello, thank you for your swift reply. I typed the exact same command you did at first 
`npx docusaurus swizzle @docusaurus/theme-classic TOC`, but for some weird reason it was constantly failing. But after clearing my node_modules folder, installing everything from scratch and trying it  again it seems to be working normal again.

**Reply by slorber** [reply]
weird, let me know if it happens again
